LunaMary/src/QuizLibrary.java

```diff
@@ -0,0 +1,40 @@
+import java.util.List;
+
+public class QuizLibrary {
+
+
+    private List<Quiz> quizList;
```

Comment:
Probably this attribute could be a map. That will filter the Quizzes by the ageRange